index.html

```diff
@@ -222,12 +220,36 @@ <h3>Ruby</h3>
                     </li>
                 </ul>
             </div>
-           
+
             <button type="button">See Project </button>
-           
+
         </div>
 
+
     </section>
+    <footer class="f-contact">
+        <h3>CONTACT US</h3>
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that
+            needs
+            coding. I’d love to help with it</p>
+        
+        <form action="https://formspree.io/f/moqrdewo" method="post">
```

Comment:
[OPTIONAL]

- [ ] I strongly suggest you use the `footer` only for:

- Copyright data
- Links
- Contact information

Since as a rule `footer` is mainly for holding data about the website, adding form into it may easily confuse people with visual disabilities, who uses screen readers to navigate through the web page